.openpublishing.redirection.json

```diff
@@ -29,6 +29,15 @@
             "redirect_url": "/aspnet/core/publishing/visual-studio-tools-for-docker",
             "redirect_document_id": true
         },
+        {
+            "source_path": "docs/core/docker/docker-basics-dotnet-core.md",
+            "redirect_url": "/dotnet/core/docker/build-docker-netcore-container",
```

Comment:
You're repeating some words in the URL which is against our SEO rules (the original name was also bad at this but we didn't know better)

Reply:
OK. When I update that article (this week) I'll change the name.